packages/terra-slide-group/src/SlideGroup.jsx

```diff
@@ -67,9 +69,15 @@ class SlideGroup extends React.Component {
       exit: transitions.exit,
       exitActive: transitions['exit-active'],
     };
+    const theme = this.context;
+    const SlideGroupClass = classNames(cx(
```

Comment:
Nit:

This variable shouldn't start uppercase

Reply:
Fixed e358850